gallery/src/demos/demo-hui-button-card.js

```diff
@@ -0,0 +1,91 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../components/demo-cards.js';
+
+const CONFIGS = [
```

Comment:
Have a look at the entities card to see how you can provide a `hass` object that will update the states when we toggle https://github.com/home-assistant/home-assistant-polymer/blob/master/gallery/src/demos/demo-hui-entities-card.js#L8